doc/contribute/build_setup.rst

```diff
@@ -11,7 +11,12 @@ Then, to quickly set up the dependencies necessary to build CCF itself and CCF a
 .. code-block:: bash
 
     $ cd <ccf_path>/getting_started/setup_vm
-    $ ./run.sh ccf-dev.yml
+    # For SGX machines
```

Comment:
Can't apply a suggestion on the whole block, but can we use the per-platform `.. tab:: SGX` pattern that we have in `install_bin.rst` for this?

Reply:
You can if you click on the +/- line and drag down in fact.

Reply:
But it needs to start from line 11, which is outside the changes so can't be suggested.

Reply:
Yeah, so it seems.